README.md

```diff
@@ -49,6 +49,25 @@ jail.create("11.1-RELEASE")
 
 Libiocage comes bundles with a CLI tool called `ioc`. It is inspired by the command line interface of [iocage](https://github.com/iocage/iocage) but meant to be developed along with the library and to spike on new features.
 
+### Custom Release (e.g. running -CURRENT)
```

Comment:
so from what i gather, this is a way of customizing a release, a.k.a: creating something that we previously might have called a template!

Reply:
I was unsure whether to make it feature of iocage or document how to do manually. This seemed to be the simplest to allow template-ish acrobatics.

Reply:
if we add it as feature, it's the officially encouraged method, if we document it, people will keep asking us how to it.
if we make the feature general enough, people can build on top of it, if we don't offer anything other than documentation, people will keep reinventing wheels that we'll have to support